Components/llsp.c

```diff
@@ -139,8 +140,10 @@ double llsp_predict(llsp_t *restrict llsp, const double *restrict metrics)
 void llsp_dispose(llsp_t *restrict llsp)
 {
 	const size_t index_last = llsp->good.columns - 1;
-	
-	free(llsp->good.matrix[index_last]);
+	// fprintf(stderr, "good.matrix = %p, index_last = %lu\n", llsp->good.matrix, index_last);
```

Comment:
This is a comment left from "printf-debugging" now it serves no purpose. Remove it.